tests/suites/test_suite_psa_crypto.function

```diff
@@ -294,6 +294,18 @@ exit:
     ((void) 0)
 #endif /* MBEDTLS_PSA_CRYPTO_SE_C */
 
+uint64_t parse_hex_string(data_t *hex_string)
```

Comment:
The name of the function and of the parameter are misleading. This is not a hex string, it's binary data. It's written in hex in the .data file, but this function doesn't receive hex.